pkg/certificate/certificate.go

```diff
@@ -193,10 +193,12 @@ type Request struct {
 	FetchPrivateKey bool
 	/*	Thumbprint is here because *Request is used in RetrieveCertificate().
 		Code should be refactored so that RetrieveCertificate() uses some abstract search object, instead of *Request{PickupID} */
-	Thumbprint   string
-	Timeout      time.Duration
-	CustomFields []CustomField
-	Location     *Location
+	Thumbprint              string
+	Timeout                 time.Duration
+	CustomFields            []CustomField
+	Location                *Location
+	ExpirationDateAttribute int
```

Comment:
This should be called `ValidityHours` as it was originally.

Reply:
reverted back the changes naw is called as before, "ValidityHours"